CMakeLists.txt

```diff
@@ -97,18 +116,18 @@ endif (NOT CMAKE_CXX_FLAGS)
 if (NOT CMAKE_BUILD_TYPE)
     # Use debug mode if building in dbg or debug directory.
     get_filename_component(_cmpvar ${CMAKE_BINARY_DIR} NAME)
-    if(_cmpvar STREQUAL "dbg" OR _cmpvar STREQUAL "debug")
+    if("_cmpvar" STREQUAL "dbg" OR "_cmpvar" STREQUAL "debug")
```

Comment:
these quotes where removed in gh-110, are they needed again or is this unintended?